src/hdr_histogram.c

```diff
@@ -88,6 +88,10 @@ static void update_min_max(struct hdr_histogram* h, int64_t value) {
                                                       value,
                                                       memory_order_relaxed,
                                                       memory_order_relaxed)) {
+#ifdef _MSC_VER
```

Comment:
Suggest you instead fix this in the definition of `atomic_compare_exchange_weak_explicit()` for Windows - at the moment you're relying on every user of `atomic_compare_exchange_weak_explicit` to know it behaves differently on Windows.

Reply:
Done